### DISCUSSION TITLE
Don't make backgrounds of other windows dimmer?

### DISCUSSION THREAD

**Discussion by MoltenMonster** [catppuccin/nvim#785] [Q&A]
This is Neotree and Vuffers on the left with edgy
<img width="651" alt="CleanShot 2024-10-23 at 21 21 23@2x" src="https://github.com/user-attachments/assets/92607e68-619b-499b-a721-1e4a1e3d39b7">

**Comment by halshar** [comment]
dimming inactive windows is already disabled by default, if you want to enable it then you can control the dim percentage as follows:
```lua
dim_inactive = {
        enabled = false, -- dims the background color of inactive window
        shade = "dark",
        percentage = 0.15, -- percentage of the shade to apply to the inactive window
    },
```
